.changes/unreleased/Under the Hood-20230130-092157.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Under the Hood
+body: Adjustment to dbt-core 1.4.0
```

Comment:
Why not 1.4.1 ?

Reply:
Actually this PR is about to adjust to 1.4.1, I changed this here